lib/schemas/AuthenticationSchema.js

```diff
@@ -14,6 +14,20 @@ const RequestSchema = require('./RequestSchema');
 module.exports = makeSchema({
   id: '/AuthenticationSchema',
   description: 'Represents authentication schemes.',
+  examples: [
+    {type: 'basic', test: '$func$2$f$'},
+    {type: 'custom', test: '$func$2$f$', fields: [{key: 'abc'}]},
+    {type: 'custom', test: '$func$2$f$', connectionLabel: 'abc'},
```

Comment:
Having this example show usage like `{{inputData.someField}}` may give devs a better sense of what the string version should look like